src/components/IntegrationTest/IntegrationTestDetailsView.tsx

```diff
@@ -13,12 +13,12 @@ import {
 } from '@patternfly/react-core';
 import { ExclamationCircleIcon } from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
 import { SearchIcon } from '@patternfly/react-icons/dist/js/icons';
-import { useModalLauncher } from '../../../components/modal/ModalProvider';
-import { useNamespace } from '../../../utils/namespace-context-utils';
 import { IntegrationTestScenarioGroupVersionKind } from '../../models';
```

Comment:
I see `IntegrationTest`, `IntegrationTestForm` and `IntegrationTestListView`, maybe all the integration tests related components can go into one IntegrationTest folder?